src/App.jsx

```diff
@@ -20,7 +34,7 @@ export function App() {
 	 */
 	const [listToken, setListToken] = useStateWithStorage(
```

Comment:
This comment is just my curiosity rather than suggesting a change needed to move your PR forward. I'm just wondering about best practices for formatting within this component. In React I understand that we want to position state related code at the top, so my instinct is that lines 35-38 would be moved essentially to line 17.  But since this is using a custom hook, I'm not sure if that changes it. Just thinking virtually out loud 🤪

Reply:
There's a lengthy comment before the state declaration. The comment could be shortened now that the default list token state is no longer active, but an explanation is still helpful for understanding how to interact with the listToken, which is more complex than it appears at first glance. 

If mentors confirm that the state declaration should come before the event handler, we can make that change.

Reply:
@patty-rose I'm pretty sure you are right about best practices and state related code. It's been moved and immediately it looked and read better. Thank you!